src/ui/SubtitleEdit.csproj

```diff
@@ -242,15 +242,6 @@
     <Content Include="Icons\VideoToggle.png" />
     <Content Include="Icons\VisualSync.png" />
     <Content Include="Icons\WaveformToggle.png" />
-    <Content Include="Resources\TesseractDictionaries.xml">
-      <SubType>Designer</SubType>
-    </Content>
-    <Content Include="Resources\HunspellBackupDictionaries.xml">
-      <SubType>Designer</SubType>
-    </Content>
-    <Content Include="Resources\HunspellDictionaries.xml">
-      <SubType>Designer</SubType>
-    </Content>
```

Comment:
These files are never part of the build, they are just used to update the `HunspellBackupDictionaries.xml.gz` and `HunspellDictionaries.xml` which are used as EmbeddedResource, so can be removed from the csproj as content.